app/config/events.php

```diff
@@ -40,8 +52,8 @@
     'database.documents.create' => [
         'description' => 'This event triggers when a database document is created.',
     ],
-    'database.documents.patch' => [
-        'description' => 'This event triggers when a database document is patched.',
+    'database.documents.update' => [
```

Comment:
Is the reasoning for `update` same as here? https://github.com/appwrite/appwrite/pull/737/files#r528358275

Reply:
Might need to re-think the naming conventions of the events name, might require extra work for migration from old setup.